pandas/io/json/normalize.py

```diff
@@ -186,7 +188,15 @@ def _pull_field(js, spec):
             for field in spec:
                 result = result[field]
         else:
-            result = result[spec]
+            # GH26284
+            try:
+                result = result[spec]
+                if not(isinstance(result, list)):
+                    # Allows collection of single object into dataframe GH26284
+                    result = [result]
+            except KeyError:
```

Comment:
We have an `errors` parameter which this ignores. I think we'll need to be aware of that here in some way, though from my mine comment I don't think we should try and tackle that here

Reply:
Not sure I understand this. The only error we can catch here is for a missing key ? Any other error would happen in the existing baseline ?

Reply:
IIUC this assumes that the user always wants to silently ignore missing keys, which is not desirable and makes for a confusing API since we have an "errors" parameter that controls that behavior for the meta

Reply:
Ah understood. If we want to give control there are clearly two ways....
(i) redefine `errors = 'ignore'` to cover both `meta` and `record path`
(ii) introduce another error flag to differentiate between `meta` and `record path`
Is there a convention or a preference in pandas before I implement ?

Reply:
I think option one